cpp/src/codegen/filter.cc

```diff
@@ -45,10 +45,11 @@ Status Filter::Make(SchemaPtr schema, ConditionPtr condition,
   GANDIVA_RETURN_FAILURE_IF_FALSE(configuration != nullptr,
                                   Status::Invalid("configuration cannot be null"));
   static Cache<FilterCacheKey, std::shared_ptr<Filter>> cache;
-  FilterCacheKey cacheKey(schema, configuration, *(condition.get()));
-  std::shared_ptr<Filter> cachedFilter = cache.GetModule(cacheKey);
+  FilterCacheKey cache_key(schema, configuration, *(condition.get()));
+  std::shared_ptr<Filter> cachedFilter = cache.GetModule(cache_key);
   if (cachedFilter != nullptr) {
     *filter = cachedFilter;
+    std::cout << "llvm filter module cache hit : " << cache_key.ToString() << std::endl;
```

Comment:
do we want this at customer site too? if yes since couts are not coming the log.. do we want our own logger..

Reply:
couts are coming praveen, they are going to server.out.

Right now, for debugging the regression failures. We can decide later if it'll be useful in production too.